src/Compilers/CSharp/Test/Semantic/Semantics/RefLocalsAndReturnsTests.cs

```diff
@@ -3818,6 +3818,23 @@ void N()
                 Diagnostic(ErrorCode.ERR_SyntaxError, "x").WithArguments(",", "").WithLocation(10, 22));
         }
 
+        [Fact]
+        [WorkItem(28117, "https://github.com/dotnet/roslyn/issues/28117")]
+        public void AssigningRefToParameter()
+        {
+            CreateCompilation(@"
+public class C
+{
+    void M(int a, ref int b)
+    {
+        a = ref b;
```

Comment:
Do we have an existing case which covers when the parameter is `in`?

Reply:
@jaredpar We have negative tests like `RefReturnRefExpression` in the same file above. The other cases lie under the escape safety feature and we have separate tests for these.
Is there another case you have in mind?


---
In reply to: [199555789](https://github.com/dotnet/roslyn/pull/28206#discussion_r199555789) [](ancestors = 199555789)